src/main/java/org/folio/fqm/repository/EntityTypeRepository.java

```diff
@@ -122,7 +122,7 @@ private EntityTypeColumn handleSingleCheckBox(String value, String refId, String
 
     return new EntityTypeColumn()
       .name(value)
-      .dataType(new BooleanType())
+      .dataType(new BooleanType().dataType("booleanType"))
```

Comment:
Do we have this string defined as a constant somewhere that we can reuse, so that we don't have to put a hard-coded string here?